chipy/videos/15260.json

```
@@ -0,0 +1,37 @@
+{
+  "category": "ChiPy",
+  "copyright_text": "CC BY",
+  "description": "Eve and Ray embarked on a two week experiment they're calling a Learning Sprint. \r\n\r\n4 hours a day, 5 days a week over two weeks they set goals and executed on them. \r\n\r\nWhat did they learn? Did it work? What fun facts did they pick up along the way? \r\n\r\nThey'll explain in their thrilling talk for all skill levels.",
+  "duration": 2618,
+  "language": "eng",
+  "quality_notes": null,
+  "recorded": "2023-05-11T19:00:00",
+  "slug": "Learning_Sprint_An_Experiment",
+  "source_url": "https://youtu.be/JIMSp2Vqjgc",
+  "speakers": [
+    "Eve Qiao",
+    " Ray Berg"
+  ],
+  "summary": "",
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/JIMSp2Vqjgc/hqdefault.jpg",
+  "title": "Learning Sprint - An Experiment",
+  "videos": [
+    {
+      "length": null,
+      "type": "host",
+      "url": "https://youtu.be/JIMSp2Vqjgc"
+    },
+    {
+      "length": null,
+      "type": "archive",
+      "url": "https://archive.org/details/may_2023-Learning_Sprint_An_Experiment"
+    },
+    {
+      "length": null,
+      "type": "conf",
+      "url": "https://www.chipy.org/meetings/223/"
+    }
+  ],
+  "veyepar_state": 10
```

Two suggestions here (that interact because JSON is very picky about trailing commas):

1. I don't believe PyVideo does anything with the `veyepar_state` field, so we can omit it.
2. The link to the event page should be in the `related_urls` section rather than listed as another video.

```suggestion
    }
  ],
  "related_urls": [
    {
      "label": "conf",
      "url": "https://www.chipy.org/meetings/223/"
    }
  ]
```